CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Pulumiverse
+
+:+1::tada: Thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing. These are just
+guidelines, not rules, so use your best judgment and feel free to propose
+changes to this document in a pull request.
```

Comment:
We should probably talk more about how Pulumiverse as an organization treats breaking/not following these guidelines. @ringods mentioned twice the word _burden_, which I find interesting.

My thinking is if a contributor opens a pull request not following any of our guidelines, I would still be happy to accept that contribution. I would still be thankful for the person's contribution, regardless. I honestly wouldn't think they did anything wrong.

This brings me to my question; are these guidelines viewed in a way that prevents people from contributing? Do these guidelines feel like a burden?

Reply:
Not to me. But to make it clearer, you might change the word "enforce" to "encourage"? I didn't see any other imperative-style statements.